units/host_monitoring_idle_termination.go

```diff
@@ -140,7 +140,7 @@ func (j *idleHostJob) Run(ctx context.Context) {
 	// if we haven't heard from the host or it's been idle for longer than the cutoff, we should terminate
 	if communicationTime >= idleTimeCutoff || idleTime >= idleTimeCutoff {
 		j.Terminated = true
-		tjob := NewHostTerminationJob(j.env, *j.host)
+		tjob := NewHostTerminationJob(j.env, *j.host, false)
```

Comment:
Do you know why Evergreen was generating host idle termination jobs for hosts that were running tasks?

Reply:
Idle hosts that start a task after the job is created?

Reply:
yep exactly